graphene-pore/graphene-pore.ipynb

```diff
@@ -0,0 +1,167 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mbuild as mb\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from foyer import Forcefield\n",
+    "from calc_numberdensity import calc_number_density"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "water = mb.load('files/tip3p.mol2')\n",
+    "water.name = 'SOL'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "c_ff = Forcefield('files/carbon.xml')\n",
+    "spce = Forcefield('files/spce.xml')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "system = mb.recipes.GraphenePoreSolvent(pore_depth=2,side_dim=2,\n",
```

Comment:
What about a structure like

```
try:
    GraphenePore = mb.recipes.GraphenePore
except ImportError:
    import porebuilder
    GraphenePore = porebuilder.porebuilder.GraphenePore
```

although I'm sure this isn't the elegant way to try this. What's the "try twice" version of a normal try/except?

Reply:
I don't know if there is actually an elegant way to do this in Python, but Stack Overflow advised me to use nested try statements.